SIL.BuildTasks.Tests/UnitTestTasks/NUnitIntegrationTests.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2016-2022 SIL International
```

Comment:
Not sure it matters much, but if this is a new file, its copyright probably can't go back to 2016.

Reply:
It's basically a copy of the old `NUnitTests.cs` file because the tests were really integration tests.